vraptor-site/content/pt/cookbook/utilizando-angularjs-vraptor4.html

```diff
@@ -0,0 +1,406 @@
+---
+title: Utilizando AngularJS no VRaptor4
+---
+# Utilizando AngularJS no VRaptor4
+Uns dos frameworks em alta é o [AngularJS](https://angularjs.org/).
+Atualmente em sua versão 1.5.3, mas com a Angular 2 a caminho, onde haverá uma reformulação total, pois a nova versão vai se valer do ECMA2015.
+Hoje vamos criar um aplicativo simples de tarefas para exemplificar como o AngularJS pode se integrar com o VRaptor4.
```

Comment:
@angeliski could u update this intro, saying that angular 2 is already available and this cookbook covers the already widest used angular 1? 

as soon as it's done we can release this doc. Sorry about my delay here